wrapanapi/systems/rhevm.py

```diff
@@ -56,6 +56,8 @@ def _get_nic_service(self, nic_name):
         for nic in self.api.nics_service().list():
             if nic.name == nic_name:
                 return self.api.nics_service().nic_service(nic.id)
+        else:
+            raise ItemNotFound('Unable to find NicService for nic {} on {}'.format(nic_name, self))
```

Comment:
`ItemNotFound` requires two args: `name, item_type` -- if you just want to raise a general msg use `NotFoundError`